raiden-dapp/src/components/notification-panel/NotificationCard.vue

```diff
@@ -29,7 +29,7 @@
         <span
           v-if="notification.link"
           class="notification-card__content__details__link"
-          @click="linkRoute"
+          @click="linkRoute()"
```

Comment:
Don't! Parenthesis mean to execute the function **now**. Why did you added them now? :confused:

Reply:
You may want to use the previou style as a consistency decision, but these parenthesis are only evaluated at event-firing time, so both syntax are almost the same, specially if the callback doesn't receive parameters, the only difference is that the previous call would receive a `$event` param, while explicitly stating the parenthesis will call it without arguments (which may or may not be what you expect, just be aware of it).

Reply:
Oh, my bad.

Reply:
Oh sorry. My bad. But yeah, maybe it works here, but "everywhere" else not. Sorry for the trouble :see_no_evil: